src/app/Dashboard/AddCard.tsx

```diff
@@ -46,28 +46,48 @@ import {
   EmptyStateVariant,
   Form,
   FormGroup,
+  NumberInput,
   Select,
   SelectOption,
-  Stack,
-  StackItem,
+  Switch,
   Text,
+  TextArea,
+  TextInput,
   Title,
 } from '@patternfly/react-core';
-import { Wizard, WizardStep } from '@patternfly/react-core/dist/js/next';
+import {
+  CustomWizardNavFunction,
+  Wizard,
+  WizardControlStep,
+  WizardNav,
+  WizardNavItem,
+  WizardStep,
+} from '@patternfly/react-core/dist/js/next';
 import { PlusCircleIcon } from '@patternfly/react-icons';
 import { useDispatch } from 'react-redux';
 import { StateDispatch } from '@app/Shared/Redux/ReduxStore';
 import { addCardIntent } from '@app/Shared/Redux/DashboardConfigActions';
-import { DashboardCards, getConfigByTitle } from './Dashboard';
+import { DashboardCards, getConfigByTitle, PropControl } from './Dashboard';
 
 interface AddCardProps {}
 
 export const AddCard: React.FunctionComponent<AddCardProps> = (props: AddCardProps) => {
   const [showWizard, setShowWizard] = React.useState(false);
   const [selection, setSelection] = React.useState('');
+  const [propsConfig, setPropsConfig] = React.useState({});
   const [selectOpen, setSelectOpen] = React.useState(false);
   const dispatch = useDispatch<StateDispatch>();
 
+  React.useEffect(() => {
```

Comment:
Looks like its called for each selection to set default. Could we move this into `handleSelect` so that state changes are batched?